### PR TITLE
Provide .gitattributes file

### PR DESCRIPTION
**Pull request by tmotyl**
to make composer install smaller and ensure correct line endings on checkout.

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All automated tests passed successfully (all builds are green)
 - [ ] Add yourself to contributors list

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Make packagist / composer download smaller
+/.ddev export-ignore
+/.github export-ignore
+/dev
```
Comment (fballiano) [inline .gitattributes:4]:
```suggestion
/dev
/docs
```

what do you think about adding the /docs folder?

Comment (elidrissidev) [reply .gitattributes:4]: Yeah, no one needs docs inside vendor

**Review by ADDISON74 — APPROVED** [review]

**Comment by tmotyl** [comment]
a little followup https://github.com/OpenMage/magento-lts/pull/3320

**Comment by sreichel** [comment]
With this change ddev commands are not available for composer installations. Also config files for code quality tools are not there.

Can we partially revert it?

**Pull request merged**